File: 21-merge-two-sorted-lists/merge-two-sorted-lists.py
# Definition for singly-linked list.
# class ListNode(object):
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next
class Solution(object):
    def mergeTwoLists(self, list1, list2):
        l=list1
        l1=list2
        f=[]
        while l:
            f.append(l.val)
            l=l.next
        while l1:
            f.append(l1.val)
            l1=l1.next
        f.sort()
        c=ListNode(0)
        d=c
        for i in f:
            d.next=ListNode(i)
            d=d.next

        return c.next
        

File: 21-merge-two-sorted-lists/merge-two-sorted-lists.java
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     int val;
 *     ListNode next;
 *     ListNode() {}
 *     ListNode(int val) { this.val = val; }
 *     ListNode(int val, ListNode next) { this.val = val; this.next = next; }
 * }
 */
class Solution {
    public ListNode mergeTwoLists(ListNode list1, ListNode list2) {
        ListNode l1=list1;
        ListNode l2=list2;
        ArrayList<Integer> f=new ArrayList<>();
        while(l1!=null){
            f.add(l1.val);
            l1=l1.next;
        }
        while(l2!=null){
            f.add(l2.val);
            l2=l2.next;
        }
        Collections.sort(f);
        ListNode h=new ListNode(0);
        ListNode head=h;
        for(int i:f){
            head.next=new ListNode(i);
            head=head.next;
        }
        return h.next;

    }
}